logfire/_internal/config.py

```diff
@@ -383,6 +383,14 @@ def _load_configuration(
             # This is particularly for deserializing from a dict as in executors.py
             pydantic_plugin = PydanticPlugin(**pydantic_plugin)  # type: ignore
         self.pydantic_plugin = pydantic_plugin or param_manager.pydantic_plugin
+        if self.pydantic_plugin.record != 'off':
+            import pydantic
+
+            # setuptools is a dependency of opentelemetry-instrumentation, we use it instead of adding "packaging" as a dependency.
+            from setuptools._vendor.packaging.version import Version
```

Comment:
I'd like it if we had a util function which tried to import things from `packaging` and then fell back to `setuptools._vendor.packaging`, so that we use the proper module where possible. `instrument_psycopg` could then also use that and we could remove the `packaging` dep from the extra.